site/content/docs/5.3/components/list-group.md

```diff
@@ -101,42 +101,45 @@ Add `.list-group-flush` to remove some borders and rounded corners to render lis
 Add the `.list-group-numbered` modifier class (and optionally use an `<ol>` element) to opt into numbered list group items. Numbers are generated via CSS (as opposed to a `<ol>`s default browser styling) for better placement inside list group items and to allow for better customization.
 
 Numbers are generated by `counter-reset` on the `<ol>`, and then styled and placed with a `::before` pseudo-element on the `<li>` with `counter-increment` and `content`.
-
+<!-- Boosted mod: add d-flex on numbered list-group-item -->
 {{< example >}}
 <ol class="list-group list-group-numbered">
-  <li class="list-group-item">A list item</li>
-  <li class="list-group-item">A list item</li>
-  <li class="list-group-item">A list item</li>
+  <li class="list-group-item d-flex">A list item</li>
+  <li class="list-group-item d-flex">A list item</li>
+  <li class="list-group-item d-flex">A list item</li>
 </ol>
 {{< /example >}}
+<!-- End mod -->
 
 These work great with custom content as well.
 
+<!-- Boosted mod: change the font-weight class and the margin-start -->
 {{< example >}}
 <ol class="list-group list-group-numbered">
   <li class="list-group-item d-flex justify-content-between align-items-start">
-    <div class="ms-2 me-auto">
-      <div class="fw-bold">Subheading</div>
-      Content for list item
+    <div class="me-auto">
+      Subheading
+      <div class="fw-normal">Content for list item</div>
     </div>
     <span class="badge bg-primary rounded-pill">14</span>
```

Comment:
If the Pr is supposed to fix #1812 the badge shouldn't be blue?

Reply:
Hmm I wasn't even aware that this PR should fix the issue 🤔 😄 I'll try to make the point on this

Reply:
It was a question because there was a reference to this issue in the opening comment of the PR.
But this issue seems to be on-hold while waiting for feedbacks from the design team so I was not sure if it should fix it or not. ;)

Reply:
Yep I haven't added it myself but yeah tried to tackle it in the last commit

Reply:
Looks god.
But I am curently wondering if it 'a good idea to do it in this PR. It may be a good idea to tackle it in a distinct one. Especially because we are still wainting for feedbacks from the design team and a11y experts (contrast for small texts is pobably too low).

Reply:
Easy for me to remove, tell me what you think better here !